R/graph.curve.params.R

```diff
@@ -151,16 +155,30 @@ graph.curve.params <- function( # nolint: object_name_linter
           group = interaction(across(all_of(group_vars)))
         )
 
-      plot1 <-
-        plot1 +
-        geom_line(
-          data = sc_to_graph,
-          alpha = alpha_samples,
-          aes(
-            color = .data$chain |> factor(),
-            group = .data$group
+      if (chain_color) {
+        plot1 <-
+          plot1 +
+          geom_line(
+            data = sc_to_graph,
+            alpha = alpha_samples,
+            aes(
+              color = .data$chain |> factor(),
```

Comment:
I think we can avoid the if-else with this approach:

```suggestion
              color = if (chain_color) .data$chain |> factor(),
```

here, color = "black" is default so the `else` doesn't need to specified.

Reply:
@d-morrison I changed the whole chunk of this code. Am I correctly following your suggestion?

Reply:
@Kwan-Jenny not quite; there's a simpler approach. revert that commit and start a teams call and I'll show you